vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -48,38 +48,48 @@ ent bundler from-git -r https://github.com/nshaw/customerportal-wip-bundle.git -
 ``` bash
 ent bundler from-git -r https://github.com/nshaw/customerportal-content-bundle.git -d | ent kubectl apply -n entando -f - 
 ```
-2. Log into the App Builder.
+2. Log into the App Builder. 
 
-3. Go to `Repository` from the left navigation bar. Two Customer Portal bundles will be visible there. `Install` the latest version of both bundles.
+3. Go to the `Repository` from the left sidebar. Two Customer Portal bundles will be visible there. `Install` the latest version of both bundles.
 
-4. To navigate to your Portal: 
+4. To navigate to your CP: 
    * From the sidebar →  `Page` → `Management` 
-   * Find the `Customer Portal` page
+   * Find the `Customer Portal` folder
    * From the `Actions` pull-down menu →  `View Published Page`
 
 ## Configuration
 ### Administrators
-In order to configure the Customer Portal and its users, the administrator will need Jira Service Management and Entando Identity Management System credentials. The administrator can create and configure users and groups, assign roles and projects, and track service tickets.  They can also customize the ticketing system. 
+In order to configure the Customer Portal and its users, the administrator will need Jira Service Management and Entando Identity Management System credentials. In this step, the admin connects the CP to Jira and customizes its features. 
```

Comment:
is it one step? sounds more like multiple steps / a procedure. is the procedure made explicit anywhere?